Pipfile

```diff
@@ -8,7 +8,9 @@ bandit = "*"
 black = "==19.10b0"
 isort = "*"
 pytest = "*"
+pytest-cov="*"
 pylint = "*"
+coveralls = "*"
```

Comment:
Updates to the `Pipfile` should always be accompanied by a new `Pipfile.lock`
Run `pipenv lock` to regenerate a new lockfile.